zeppelin-web/src/app/notebook/paragraph/paragraph-results.html

```diff
@@ -23,9 +23,19 @@
        ng-bind-html="paragraph.result.comment">
   </div>
 
-  <div id="p{{paragraph.id}}_text"
-       class="text"
-       ng-if="getResultType() == 'TEXT'"></div>
+  <div id="{{paragraph.id}}_text"
+       ng-if="getResultType() == 'TEXT'">
+    <div class="fa fa-level-down scroll-paragraph-down"
+         ng-show="showScrollDownIcon()"
+         ng-click="scrollParagraphDown()" ></div>
+    <div id="p{{paragraph.id}}_text"
+         style="max-height: {{paragraph.config.graph.height}}px; overflow: auto"
+         ng-scroll="someFunction()"
```

Comment:
What is that line for? I didn't see any change when trying with and without.
Not sure if its necessary since `someFunction` doesn't exist anywhere